io.github.mpc_qt.Mpc-Qt.yml

```diff
@@ -0,0 +1,167 @@
+---
+app-id: io.github.mpc_qt.Mpc-Qt
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: mpc-qt
+rename-icon: mpc-qt
+rename-desktop-file: mpc-qt.desktop
+copy-icon: true
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--socket=pulseaudio'
+  - '--device=dri'
+  - '--filesystem=host'
```

Comment:
this is also very broad permissions.

Reply:
We can probably drop the wayland and dri permissions since we're forcing X11 in main.cpp. This manifest was based on a similar app for what it's worth. Thanks for the feedback.

Reply:
I've done some more testing and we need dri for libGL. We need filesystem=host to save screenshots.

Reply:
> We need filesystem=host to save screenshots.

Are you sure? this is the wide permission I was talking about.